### PR TITLE
Routes Update

### PR DESCRIPTION
**Pull request by cjkelling**
Change routes from hand rolled to resource routes. Also ran RuboCop to fix any errors.

Co-Authored-By: Laura Schulz <lrs8810@users.noreply.github.com>
Co-Authored-By: Michael Cooper <CoopTang@users.noreply.github.com>
Co-Authored-By: Scott Regenthal <freeheeling@users.noreply.github.com>

### CODE REVIEW

**Comment by freeheeling** [comment]
Appreciate you walking us all through the process.

File: app/controllers/admin/dashboard_controller.rb [hunk 1]
```diff
@@ -1,4 +1,5 @@
+# frozen_string_literal: true
+
 class Admin::DashboardController < ApplicationController
-  def index
-  end 
+  def index; end
```
Comment (lrs8810) [inline app/controllers/admin/dashboard_controller.rb:4]: I really like this syntax!

File: app/controllers/application_controller.rb [hunk 2]
```diff
@@ -14,14 +14,14 @@ def current_user
   end
 
   def merchant_employee?
-    current_user && current_user.merchant_employee?
+    current_user&.merchant_employee?
```
Comment (lrs8810) [inline app/controllers/application_controller.rb:17]: I really like this syntax! It cleaned up this line well and is still readable.

File: config/routes.rb [hunk 3]
```diff
@@ -1,54 +1,35 @@
...
-
-  get '/items/:item_id/reviews/new', to: 'reviews#new'
-  post '/items/:item_id/reviews', to: 'reviews#create'
-
-  get '/reviews/:id/edit', to: 'reviews#edit'
-  patch '/reviews/:id', to: 'reviews#update'
-  delete '/reviews/:id', to: 'reviews#destroy'
+  resources :items, except: %i[new create] do
+    resources :reviews, only: %i[new create]
+  end
```
Comment (lrs8810) [inline config/routes.rb:6]: Great job!

**Review by lrs8810 — APPROVED** [review]

**Pull request merged**